doc/src/manual/style-guide.md

```diff
@@ -123,7 +123,7 @@ Types such as `Union{Function,AbstractString}` are often a sign that some design
 When creating a type such as:
 
 ```julia
-type MyType
+struct MyType
```

Comment:
if I were at a computer right now, yes. `mutable struct` would be better here since that's the situation where union types fields is more likely to be a problem